starwars/src/index.css

```diff
@@ -2,6 +2,6 @@ body {
   margin: 0;
   padding: 0;
   font-family: sans-serif;
-  /*background-image: url('./sw-bg.jpg');*/
+  background-image: url('./sw-bg.jpg');
   background-size: cover;
```

Comment:
- Great job on the sprint challenge Haja. 
- Your project is styled properly and design looks good. It is fully responsive and looks great on both mobile and tablet views 
- You successfully completed the MVP by fetching data from 3rd party API and displayed it on the webpage. You have properly split functionality into smaller components and passed necessary data as props.

Your code is clean and Your git workflow is fluid.
Overall:
* Able to fetch and display data (MVP)
* Clean and modular code with multiple components
* Implemented styling
* Responsive design

Possible Improvements:
* Add some kind of loading icon or information that data is being fetched. After loading Your app it takes some time to load and render everything so it would be great fit here